tests/test_dates.py

```diff
@@ -0,0 +1,222 @@
+            assert dates['closing_date'] == datetime.datetime(2016, 7, 20, 23, 59, 59)
+
+    def test_get_publishing_dates_for_two_week_briefs_are_correct_if_published_on_a_thursday(self):
+        with mock.patch('dmutils.dates.datetime') as mock_date:
+            mock_date.utcnow.return_value = datetime.datetime(2016, 7, 7, 15, 21, 39, 417900)
+            brief = {
+                'requirementsLength': '2 weeks',
+            }
+
+            dates = dates_package.get_publishing_dates(brief)
+
+            assert dates['questions_close'] == datetime.datetime(2016, 7, 14, 23, 59, 59)
+            assert dates['answers_close'] == datetime.datetime(2016, 7, 20, 23, 59, 59)
+            assert dates['closing_date'] == datetime.datetime(2016, 7, 21, 23, 59, 59)
+
+    def test_get_publishing_dates_for_two_week_briefs_are_correct_if_published_on_a_friday(self):
+        with mock.patch('dmutils.dates.datetime') as mock_date:
+            mock_date.utcnow.return_value = datetime.datetime(2016, 7, 8, 15, 21, 39, 417900)
+            brief = {
+                'requirementsLength': '2 weeks',
+            }
+
+            dates = dates_package.get_publishing_dates(brief)
+
+            assert dates['questions_close'] == datetime.datetime(2016, 7, 15, 23, 59, 59)
+            assert dates['answers_close'] == datetime.datetime(2016, 7, 21, 23, 59, 59)
+            assert dates['closing_date'] == datetime.datetime(2016, 7, 22, 23, 59, 59)
+
+    def test_get_publishing_dates_for_two_week_briefs_are_correct_if_published_on_a_saturday(self):
+        with mock.patch('dmutils.dates.datetime') as mock_date:
+            mock_date.utcnow.return_value = datetime.datetime(2016, 7, 9, 15, 21, 39, 417900)
+            brief = {
+                'requirementsLength': '2 weeks',
+            }
+
+            dates = dates_package.get_publishing_dates(brief)
+
+            assert dates['questions_close'] == datetime.datetime(2016, 7, 15, 23, 59, 59)
+            assert dates['answers_close'] == datetime.datetime(2016, 7, 22, 23, 59, 59)
+            assert dates['closing_date'] == datetime.datetime(2016, 7, 23, 23, 59, 59)
+
+    def test_get_publishing_dates_for_two_week_briefs_are_correct_if_published_on_a_sunday(self):
+        with mock.patch('dmutils.dates.datetime') as mock_date:
+            mock_date.utcnow.return_value = datetime.datetime(2016, 7, 10, 15, 21, 39, 417900)
+            brief = {
+                'requirementsLength': '2 weeks',
+            }
+
+            dates = dates_package.get_publishing_dates(brief)
+
+            assert dates['questions_close'] == datetime.datetime(2016, 7, 15, 23, 59, 59)
+            assert dates['answers_close'] == datetime.datetime(2016, 7, 22, 23, 59, 59)
+            assert dates['closing_date'] == datetime.datetime(2016, 7, 24, 23, 59, 59)
+
+    def test_get_publishing_dates_returns_correct_dates_if_brief_is_published_with_no_requirementLength(self):
```

Comment:
@idavidmcdonald Yep, have added a couple more tests to cover the possibilities of published briefs and the different states requirementsLength could be in.